iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/webservice/auth/IUserManagementClient.java

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
in dotnet we put UserManagementClient in services/external, any reason for the change?